lusSTR/tests/test_snps.py

```diff
@@ -239,3 +239,23 @@ def test_snp_bins(tmp_path):
     for snp_set in range(0, 10):
         path = tmp_path / f"evidence_samples/Kin_pos_1ng_snp_evidence_set{snp_set}.csv"
         assert path.is_file()
+
+
+def test_uas_version5(tmp_path):
+    input_sample = data_file("NA24385 Sample Report 2023_09_07_15_11_11.xlsx")
+    exp_out = data_file("kinsnps/uasversion_snp_evidence.csv")
+    obs_out = str(tmp_path / "kin_v5_snps_evidence.csv")
+    arglist = [
+        "config",
+        "-w",
+        str(tmp_path),
+        "-o",
+        "kin_v5",
+        "--input",
+        input_sample,
+        "--snps",
+        "--kintelligence",
+    ]
+    lusSTR.cli.main(lusSTR.cli.get_parser().parse_args(arglist))
+    all_arglist = ["snps", "all", "-w", str(tmp_path)]
+    lusSTR.cli.main(lusSTR.cli.get_parser().parse_args(all_arglist))
```

Comment:
It looks like this test isn't checking the output in any way. It's just making sure the code runs without exploding? (I call these "smoke tests" and use them frequently. 😎)